generators/dotnetstandard-client/PrincipleStudios.OpenApiCodegen.Client.Analyzers/templates/client.handlebars

```diff
@@ -50,12 +50,12 @@ namespace {{packageName}}
                         .Replace("{" + "{{rawName}}" + "}", global::System.Web.HttpUtility.UrlEncode({{#if required}}{{paramName}}{{else}}global::PrincipleStudios.OpenApiCodegen.Json.Extensions.Optional.GetValueOrThrow({{paramName}}){{/if}}.ToString())){{/if}}{{/each}});
 {{/if}}
 {{#if requestBody.isForm}}
-            requestMessage.Content = new global::System.Net.Http.MultipartFormDataContent()
+            requestMessage.Content = new global::System.Net.Http.FormUrlEncodedContent(new global::System.Collections.Generic.Dictionary<string, string>
```

Comment:
This is a bug that I fixed as part of this work: Apparently, `MultipartFormDataContent` was never the right choice and was rejected from MVC with a 415 error. There's a lot of work to be done to make this all function correctly, but this was a start for the basic examples. See #100 for more details.